ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+- [ ] I have searched the [issues](
```

Comment:
`I have checked that the repository is not registered to Issuehunt yet and there are not duplicated requests for it.`